src/pages/components/tabs/usage.mdx

```diff
@@ -323,23 +336,32 @@ tab to the edge of the content area.
 For further content guidance, see Carbon's
 [content guidelines](/guidelines/content/overview).
 
-## Behaviors
+## Universal behaviors
 
 ### States
 
-Tabs allow for three states: **selected, unselected,** and **disabled**. The
-default view is that one tab is preselected and is usually the first tab. Only
-one tab can be selected at a time. When a user chooses a new item, the previous
-tab is automatically deselected. If a user navigates away from a tab, a user
-should return to the last tab selected.
+Tabs have two states—**selected,** and **unselected,**. Other interactive states
+are **hover, focus,** and **disabled**. The default view is one tab is
+preselected and is usually the first tab. Only one tab can be selected at a
+time. When a user chooses a new item, the previous tab is automatically
+deselected. If a user navigates away from a tab, a user should return to the
```

Comment:
>  If a user navigates away from a tab, a user should return to the last tab selected.

Michael suggested changing this sentence to say this "If a user navigates away from a tab, it remains the selected tab until altered by the user." You can make the call if you want to change it or not to something like this.